crates/forge/bin/cmd/test/mod.rs

```diff
@@ -604,6 +612,47 @@ fn list(
     Ok(TestOutcome::new(BTreeMap::new(), false))
 }
 
+/**
+ * Creates table summary header
+ */
+fn create_test_summary_table_header(summary_table: &mut Table) -> Row {
+    summary_table.apply_modifier(UTF8_ROUND_CORNERS);
+    Row::from(vec![
+        Cell::new("Test Suites")
+            .set_alignment(CellAlignment::Center)
+            .add_attribute(Attribute::Bold),
+        Cell::new("Passed")
+            .set_alignment(CellAlignment::Center)
+            .add_attribute(Attribute::Bold)
+            .fg(Color::Green),
+        Cell::new("Failed")
+            .set_alignment(CellAlignment::Center)
+            .add_attribute(Attribute::Bold)
+            .fg(Color::Red),
```

Comment:
I know we already showed red, but I'd avoid showing any red unless we actually have a failure

Reply:
Good idea. Should we do the same for all 3 columns or only for failures?

Reply:
Hmm good question. What do other frameworks do? I don't want to overcomplicate it, mainly just don't like that we currently always show red—it looks bad at a glance since everyone associates red=bad